platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/EventCreationConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.event.tipset;
+
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.ConfigProperty;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.time.Duration;
+
+/**
+ * Configuration for event creation.
+ *
+ * @param useTipsetAlgorithm             if true, use the tipset event creation algorithm
+ * @param maxCreationRate                the maximum rate (in hz) that a node can create new events. The maximum rate
```

Comment:
Measuring it this way is not very simple and seems prone to errors. Why not make it a value between 0 and 1, or 0 and 100, and use that to scale to the size of the network?

Reply:
I'm not sure I understand what you mean by this, let's discuss.

Reply:
We discussed this, closing.